_data/routingrules.json

```diff
@@ -888,7 +888,8 @@
   "^/author/bernhard-rosenkranzer(/?|/index.html)$ /blog/ [R,L,NC]",
   "^/author/baolinwang(/?|/index.html)$ /author/baolin-wang/ [R,L,NC]",
   "^/author/arndbergmann(/?|/index.html)$ /author/arnd-bergmann/ [R,L,NC]",
-  "^/assets/pdf/Linaro_Training_Catalogue_January_2022.pdf$ /assets/pdf/Linaro_Training_Catalogue_February_2023.pdf [R,L,NC]",
+  "^/assets/pdf/Linaro_Training_Catalogue_January_2022.pdf$ /assets/pdf/Linaro_Training_Catalogue_January_2024.pdf [R,L,NC]",
+  "^/assets/pdf/Linaro_Training_Catalogue_February_2024.pdf$ /assets/pdf/Linaro_Training_Catalogue_January_2024.pdf [R,L,NC]",
```

Comment:
Should this be February_2023, not February_2024?

Reply:
Good spot.

I think I was concentrating so hard to make sure I spelled February correctly that I overlooked the obvious. Thanks.

(fixed in most recent push)